completion/structured_output_cookbook.ipynb

```diff
@@ -0,0 +1,249 @@
+    "```json\n",
+    "response_format={\n",
+    "    \"type\": \"json_schema\",\n",
+    "    \"json_schema\": {\n",
+    "        \"schema\": {\n",
+    "            \"type\": \"object\",\n",
+    "            \"properties\": {\n",
+    "                \"title\": {\"type\": \"string\"},\n",
+    "                \"genre\": {\"type\": \"string\"},\n",
+    "                \"year\": {\"type\": \"number\"},\n",
+    "                \"rating\": {\"type\": \"number\"},\n",
+    "                \"streaming_platforms\": {\n",
+    "                    \"type\": \"array\",\n",
+    "                    \"items\": {\"type\": \"string\"}\n",
+    "                }\n",
+    "            },\n",
+    "            \"required\": [\"title\", \"genre\", \"year\", \"rating\", \"streaming_platforms\"]\n",
+    "        }\n",
+    "    }\n",
+    "}\n",
+    "```\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "591455ef-6816-43df-9984-75f57691ab27",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "system_message = \"You are a helpful movie recommendation assistant.\"\n",
+    "user_message = \"Please recommend the most interesting movie in the world.\"\n",
+    "\n",
+    "response_format = {\n",
+    "        \"type\": \"json_schema\",\n",
+    "        \"json_schema\": {\n",
+    "            \"schema\": {\n",
+    "                \"type\": \"object\",\n",
+    "                \"properties\": {\n",
+    "                    \"title\": {\"type\": \"string\"},\n",
+    "                    \"genre\": {\"type\": \"string\"},\n",
+    "                    \"year\": {\"type\": \"number\"},\n",
+    "                    \"rating\": {\"type\": \"number\"},\n",
+    "                    \"streaming_platforms\": {\"type\": \"array\", \"items\": {\"type\": \"string\"}}\n",
+    "                },\n",
+    "                \"required\": [\"title\", \"genre\", \"year\", \"rating\", \"streaming_platforms\"]\n",
+    "            }\n",
+    "        }\n",
+    "    }"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a91b3634-70d6-4ac9-80d4-968ac721fa56",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "response = client.chat.chat(\n",
+    "    model=\"palmyra-x5\",\n",
+    "    messages=[\n",
+    "        {\"role\": \"system\", \"content\": system_message},\n",
+    "        {\"role\": \"user\", \"content\": user_message}\n",
+    "    ],\n",
+    "    response_format=response_format\n",
+    ")\n",
+    "\n",
+    "print(json.loads(response.choices[0].message.content))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d024ff8f-0bb6-4c73-a775-0ac2586d4340",
+   "metadata": {},
+   "source": [
+    "## Using Pydantic Objects and `chat.parse`\n",
+    "\n",
+    "Instead of defining a JSON Schema manually, you can use **Pydantic models** to define the structure of the AI’s response. The `chat.parse` helper validates and converts the response into a Python object.\n"
```

Comment:
```suggestion
    "Instead of defining a JSON Schema manually, you can use [Pydantic models](https://docs.pydantic.dev/latest/) to define the structure of the AI’s response. The `chat.parse` helper validates and converts the response into a Python object.\n"
```